etc/schema.json

```diff
@@ -3823,6 +3823,9 @@
                                 },
                                 "tls": {
                                     "$ref": "#/$defs/stats_applayer_error"
+                                },
+                                "websockets": {
```

Comment:
rfc 6455 refers to it as "WebSocket" (singular), so lets use singular in our code and docs

Reply:
indeed